integration_tests/test_str_attributes.py

```diff
@@ -25,6 +25,17 @@ def strip():
     assert "     AASAsaSas    " .lstrip() == "AASAsaSas    "
     assert "     AASAsaSas    " .strip() == "AASAsaSas"
 
+def find():
+    s: str
+    sub: str
+    s = "AaaaAABBbbbbBB!@12223BN"
+    sub = "@"
```

Comment:
Test for sub-strings of length more than 1? And probably empty sub-string as well?

Reply:
I added them